build.properties

```diff
@@ -8,12 +8,12 @@ metacat.version=3.0.0
 metacat.releaseCandidate=
 
 # Tomcat dev deployment directory
-build.tomcat.dir=/usr/share/tomcat8
+build.tomcat.dir=/usr/share/tomcat9
```

Comment:
Can we delete this altogether?? I could be wrong, but I don't believe this property is necessary any more? 

On my dev machine, I have not changed it from the original setting (`build.tomcat.dir=/usr/share/tomcat8`), and that path does not actually exist on my laptop, yet I've been successfully building metacat and running tests for 11 months :-) 

I looked through `build.xml`, and when I follow through the steps that use `build.tomcat.dir`, it looks like it's used in order to add all the jar files form the tomcat `lib` directory into the classpath. However, now we're using maven to manage dependencies, it seems like the compiler should find everything it needs in the lib/maven/ directory?

Let me know if I missed something